### PR TITLE
Move client configuration to backend

### LINKED ISSUE #281: Combine client and server config
We should combine the clients .env file and the servers config toml-file into one config file and let the client get its config from the server.

### PR DESCRIPTION
**Pull request by koplas**
Closes #281

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -144,6 +153,16 @@ type Sources struct {
 	DefaultMessage string `toml:"default_message"`
 }
 
+// Client are the config options for the client.
+type Client struct {
+	KeycloakUrl      string `toml:"keycloak_url"`
+	KeycloakRealm    string `toml:"keycloak_realm"`
```
Comment (s-l-teichmann) [inline pkg/config/config.go:159]: If you add json struct tags you can simply send the structure as JSON in the endpoint.

**Review by s-l-teichmann — CHANGES_REQUESTED** [review]

File: pkg/config/config.go (hunk 1 above)
Comment (koplas) [reply pkg/config/config.go]: Done

File: pkg/config/config.go [hunk 2]
```diff
@@ -313,6 +341,12 @@ func (cfg *Config) fillFromEnv() error {
 		envStore{"ISDUBA_TEMP_STORAGE_FILES_USER", storeInt(&cfg.TempStore.FilesUser)},
 		envStore{"ISDUBA_TEMP_STORAGE_DURATION", storeDuration(&cfg.TempStore.StorageDuration)},
 		envStore{"ISDUBA_SOURCES_DEFAULT_MESSAGE", storeString(&cfg.Sources.DefaultMessage)},
+		envStore{"ISDUBA_CLIENT_KEYCLOAK_URL", storeString(&cfg.Client.KeycloakURL)},
+		envStore{"ISDUBA_CLIENT_KEYCLOAK_REALM", storeString(&cfg.Client.KeycloakRealm)},
+		envStore{"ISDUBA_CLIENT_KEYCLOAK_CLIENTID", storeString(&cfg.Client.KeycloakClientID)},
```
Comment (s-l-teichmann) [inline pkg/config/config.go:346]: s/ISDUBA_CLIENT_KEYCLOAK_CLIENTID/ISDUBA_CLIENT_KEYCLOAK_CLIENT_ID/

File: pkg/config/config.go [hunk 3]
```diff
@@ -144,6 +153,16 @@ type Sources struct {
 	DefaultMessage string `toml:"default_message"`
 }
 
+// Client are the config options for the client.
+type Client struct {
+	KeycloakURL      string `toml:"keycloak_url" json:"keycloak_url"`
+	KeycloakRealm    string `toml:"keycloak_realm" json:"keycloak_realm"`
+	KeycloakClientID string `toml:"keycloak_client_id" json:"keycloak_client_id"`
+	UpdateInterval   int    `toml:"update_interval" json:"update_interval"`
```
Comment (s-l-teichmann) [inline pkg/config/config.go:161]: Okay, it's for the client,  but it feels a bit off as we use time.Duration for time intervals used by the server.

**Review by s-l-teichmann — CHANGES_REQUESTED** [review]

File: pkg/config/config.go (hunk 3 above)
Comment (koplas) [reply pkg/config/config.go]: Done

**Pull request merged**